src/include/OpenImageIO/fmath.h

```diff
@@ -214,9 +214,9 @@ OIIO_FORCEINLINE uint64_t rotl64 (uint64_t x, int k) {
 
 
 /// clamp a to bounds [low,high].
-template <class T>
+template <class T, class S>
 inline T
-clamp (T a, T low, T high)
+clamp (T a, S low, S high)
```

Comment:
Those suggestions sound totally reasonable. Let me look at it, but in any case I no longer need to do this before the other thing I was really trying to submit.